gbdx_auth/gbdx_auth.py

```diff
@@ -145,16 +152,24 @@ def get_session(config_file=None):
     """Returns a requests session with gbdx oauth2 baked in.
 
     If you provide GBDX_ACCESS_TOKEN and GBDX_REFRESH_TOKEN via env vars it will
-    use those credentials. If you provide a path to a config file, it will look
-    there for the credentials.  If you don't it will try to pull the credentials
-    from environment variables (GBDX_USERNAME, GBDX_PASSWORD,
+    use those credentials. If GBDX_RUNTIME_FILE is defined with user_token in
+    json object it will use those credentials. If you provide a path to a config
+    file, it will look there for the credentials. If you don't it will try to
+    pull the credentials from environment variables (GBDX_USERNAME, GBDX_PASSWORD,
     GBDX_CLIENT_ID, GBDX_CLIENT_SECRET).  If that fails and you have a
     '~/.gbdx-config' ini file, it will read from that.
     """
 
     if os.environ.get("GBDX_ACCESS_TOKEN", None):
         return session_from_existing_token(access_token=os.environ.get("GBDX_ACCESS_TOKEN", None), refresh_token=os.environ.get("GBDX_REFRESH_TOKEN", None))
 
+    if os.path.isfile(GBDX_RUNTIME_FILE):
+        with open(GBDX_RUNTIME_FILE) as f:
+            runtime_json = json.loads(f.read())
+
+        if runtime_json.get('user_token', None):
+            return session_from_existing_token(access_token=runtime_json['user_token'])
```

Comment:
`runtime_json['user_token']` may raise key error here since user_token key/value is not always present unless the workflow user allows impersonation and the task permits it